docs/dev-environment.md

```diff
@@ -39,6 +39,14 @@ Removing the installation:
 docker compose down --volumes
 ```
 
+### Updating localizations
+
+After modifying `i18n` files, run the following to update localization caches:
+
+```sh
+docker compose exec -it mediawiki bash -c "WBS_DOMAIN=maint php w/maintenance/rebuildLocalisationCache.php --lang=en --force"
```

Comment:
```suggestion
docker compose run --rm -it mediawiki bash -c "WBS_DOMAIN=maint php w/maintenance/rebuildLocalisationCache.php --lang=en --force"
```

Would it make sense to use `run` instead of `exec` so it would also work when you don't have the stack up and running?